cmd/main_test.go

```diff
@@ -301,3 +306,33 @@ var _ = DescribeTable("TestRunAccessCheck", func(user string, namespace string,
 		"patch",
 		false),
 )
+
+var _ = DescribeTable("TestGetNamespacesWithAccess", func(allNamespaces []k8sapi.Namespace, 
+	expectedNs []k8sapi.Namespace) {
+	e := echo.New()
+	cfg, _ := config.GetConfig()
+	clientset, _ := kubernetes.NewForConfig(cfg)
+	req := httptest.NewRequest(http.MethodGet, "/", nil)
+	rec := httptest.NewRecorder()
+	c := e.NewContext(req, rec)
+	c.Request().Header.Set("X-Email", "test@example.com")
+	authCl := clientset.AuthorizationV1()
+
+	actualNs, err := getNamespacesWithAccess(e, c, authCl, allNamespaces)
+
+	Expect(actualNs).To(Equal(expectedNs))
+	Expect(err).NotTo(HaveOccurred(), "Unexpected error testing GetNamespacesWithAccess")
+},
+	Entry(
+		"when get namspace with access allows all namespaces",
+		[]k8sapi.Namespace{ns1, ns2},
+		[]k8sapi.Namespace{ns1, ns2}),
+	Entry(
+		"when get namspace with access denies all namespaces",
```

Comment:
do we have a test for a namespace with "create", "list", "watch",  but not "delete"?
or a test for a namespace with "create", "list", "watch", "delete" *and* another verb (e.g patch)?

if so, have it explained in the entry

Reply:
Added test case with namespace without delete